src/utils/supabase/queries.ts

```
@@ -0,0 +1,917 @@
+type Notification = {
+  id: string;
+  user_id: string;
+  type: string;
+  title: string;
+  message: string;
+  read: boolean;
+  created_at: string;
+  data?: Record<string, any>;
+};
+
+type QueryError = {
```

ESLint

[error] 'QueryError' is defined but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)